apps/settings/test/marionette/app/regions/screen_lock.js

```diff
@@ -69,7 +69,8 @@ ScreenLockPanel.prototype = {
   },
 
   getPasscode: function() {
-    return this.client.settings.get('lockscreen.passcode-lock.code');
```

Comment:
There are no callers anymore, so I figured why not simply remove it instead of throwing.